crypt-gpgme.c

```diff
@@ -63,11 +63,15 @@
  * Helper macros.
  */
 #define digitp(p)   (*(p) >= '0' && *(p) <= '9')
+
+// clang-format off
 #define hexdigitp(a) (digitp (a)                     \
                       || (*(a) >= 'A' && *(a) <= 'F')  \
                       || (*(a) >= 'a' && *(a) <= 'f'))
 #define xtoi_1(p)   (*(p) <= '9'? (*(p)- '0'): \
                      *(p) <= 'F'? (*(p)-'A'+10):(*(p)-'a'+10))
+// clang-format on
+
```

Comment:
that one makes the above code prettier for sure, but it's again a code that has been introduced 12 years ago, and hasn't been changed since. It's fairly clear at what it does: checking that a digit is a digit, a hex digit is a hex digit and the last one transforms an hex into an int.

So whether it's nicely formatted or not, the RHS soup isn't likely to be changed anytime soon, and if it's giving a phishy result, then you could realign it properly for the sake of double checking.

All in all, I guess we can leave it uglified by clang-format without afterthoughts.